ca_tools/action/FollowLine.action

```diff
@@ -1,10 +1,10 @@
 # Goal
-int32 amount
+int32 goal_seconds
 
 ---
 # Result
 string[] final_result
```

Comment:
I think you should return a duration msg, they are included on the ros standard. See [here](http://wiki.ros.org/ROS/Tutorials/CreatingMsgAndSrv)

Reply:
Also, add a bool for a positive or negative result